eventkitd-bigquery/bigquery/client.go

```diff
@@ -0,0 +1,214 @@
+package bigquery
```

Comment:
copyright?

Reply:
We didn't use copyright headers so far. And this repository is moved from `jtolio` namespace. Can I add a `Storj`  copyright header with current year to all files?

Reply:
yeah sorry, you can ignore this for this pr!